src/context/WalletProvider/Ledger/components/Chains.tsx

```diff
@@ -48,6 +33,8 @@ import { portfolio, portfolioApi } from 'state/slices/portfolioSlice/portfolioSl
 import { selectAssets, selectWalletChainIds } from 'state/slices/selectors'
 import { useAppDispatch, useAppSelector } from 'state/store'
 
+import { availableLedgerAppAssetIds, availableLedgerAppChainIds } from '../constants'
```

Comment:
Just moved this to a `constants.ts` file as the lists are being consumed in multiple places and we don't want them getting out of sync.